brave/brave-ios
# Fix #5283: Option to switch to an open tab

## Summary of Changes


This pull request fixes #5283

## Submitter Checklist:


## Test Plan:

For all the test enable Switch To an Open Tab setting.

Test Plan 1 External Link:

- Switch to Normal Browsing Mode
- Open an external link in Brave
- Open another link in another tab and leave it as active
- Close the browser
- Open the same external link outside the Brave
- Check if external link switches to the existing tab not open new tab with that url link

## Screenshots:

- Opening an external Link:


https://user-images.githubusercontent.com/6643505/193651030-cbea22a5-e0f6-488b-80e5-16281c7fdecf.MP4



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
